client/src/context/useMessagingContext.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useContext, createContext, FC, useEffect } from 'react';
+import { Conversation, Message } from '../interface/Message';
+import { Conversation as APIConversation } from '../interface/Messaging';
+import { fetchConversations } from '../helpers/APICalls/messaging';
+import { AuthContext } from '../context/useAuthContext';
+
+interface IMessagingContext {
+  conversations: Conversation[];
+}
+
+export const MessagingContext = createContext<IMessagingContext>({
+  conversations: [],
+});
+
+export const MessagingProvider: FC = ({ children }): JSX.Element => {
+  const { loggedInUser } = useContext(AuthContext);
+  const [conversations, setConversations] = useState<Conversation[]>([]);
```

Comment:
How are we planning on updating and displaying the messages within a conversation? Also, how are we going to differentiate which conversation is the active one?

Reply:
Added add message functionality to context, assuming that with the socket io integration the backend would update the database.  Added indicator for active conversation.
![image](https://user-images.githubusercontent.com/62892917/128221800-e48d3de5-cb7f-4b96-8126-c11f13b18af2.png)